QADI_README.md

```
@@ -0,0 +1,63 @@
+# QADI Command Line Tool
+
+A command-line tool that uses Google's Gemini API to analyze questions using the QADI (Question, Abduction, Deduction, Induction) methodology.
+
+## Setup
+
+1. Add your Google API key to `.env`:
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines

(MD032, blanks-around-lists)